openshift-ci/Dockerfile.bundle.ci

```diff
@@ -8,8 +8,8 @@ LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
 LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=performance-addon-operator
-LABEL operators.operatorframework.io.bundle.channels.v1=4.7.0
-LABEL operators.operatorframework.io.bundle.channel.default.v1=4.7.0
+LABEL operators.operatorframework.io.bundle.channels.v1=${CHANNEL}
```

Comment:
Should this be only `${CHANNEL}` ?  in this case will translate to 4.7  instead 4.7.0 we had previously ,
or should this be `${CHANNEL}.${ZVERSION}` ?
Same question applies to the rest

Reply:
Ah, now I understand, we want to use here only Major.Minor with no Z stream